x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_editor/system_prompt/index.tsx

```diff
@@ -67,7 +67,6 @@ const SystemPromptComponent: React.FC<Props> = ({
       isCleared={isCleared}
       refetchConversations={refetchConversations}
       isSettingsModalVisible={isSettingsModalVisible}
-      onSystemPromptSelectionChange={onSystemPromptSelectionChange}
```

Comment:
This was causing a race condition in our tests, and could cause that condition for users too. Providing this function updates the UI before the server has updated. This function was added for use from the assistant settings. If a conversation is selected like it is here, the update is handled by the `SelectSystemPrompt` component